darts/models/forecasting/nbeats.py

```diff
@@ -574,7 +574,6 @@ def __init__(
             The number of blocks making up every stack.
         num_layers
             The number of fully connected layers preceding the final forking layers in each block of every stack.
-            Only used if `generic_architecture` is set to `True`.
```

Comment:
Could you also change the docstring of `_NBEATSModule` ?

Reply:
Done